src/cmake/compiler_flags.cmake

```diff
@@ -26,7 +26,7 @@ else()
         set(warnings "-Wall -Wextra -Wshadow -Wno-strict-aliasing -Wold-style-cast -Wdouble-promotion")
     else()
         add_definitions(-fno-exceptions)
-        set(warnings "-Wall -Wextra -Wshadow -Wno-strict-aliasing -Wold-style-cast -Wdouble-promotion -Wformat=2 -Weffc++")
+        set(warnings "-Wall -Wextra -Wshadow -Wno-strict-aliasing -Wold-style-cast -Wdouble-promotion -Wformat=2")
```

Comment:
> This is because it throws warnings for the grpc generated headers and I
can't manage to set the SYSTEM/-isystem flag correctly for these
includes.